README.md

```diff
@@ -1,6 +1,8 @@
 # Trusted Data Format Specification
 
-Trusted Data Format (TDF) is an Open, Interoperable, JSON encoded data format for implementing Data Centric Security for objects (such as files or emails). This repository specifies the protocols and schemas required for TDF operation. Versioning of this spec follows the [Semver standard](https://semver.org/). The latest major version of the TDF spec and corresponding protocol is TDF 3.x.x.
+Trusted Data Format (TDF) is an Open, Interoperable data format for implementing Data Centric Security for objects (such as files or emails). This repository specifies the protocols and schemas required for TDF operation. Versioning of this spec follows the [Semver standard](https://semver.org/). The latest major version of the TDF spec and corresponding protocol is TDF 4.x.x.
```

Comment:
```suggestion
Trusted Data Format (TDF) is an Open, Interoperable data format for implementing Data Centric Security for objects (such as files or emails). This repository specifies the protocols and schemas required for TDF operation. Versioning of this spec follows the [Semver standard](https://semver.org/).
```

(I might have added this myself in the first place at some point but) I'd actually argue that the docs themselves maybe shouldn't referenced fixed version numbers, in order to reduce the number of places we have to check to keep them in sync.

Reply:
agreed.